Course_Part_05/src/Review01.java

```diff
@@ -0,0 +1,61 @@
+public class Review01 {
+    public static void main(String[] args) {
+        double[] doubleArray = {6.3, 7.8, 5.9};
```

Comment:
in de naamgeving hoef je eigenlijk niet aan te geven dat het om double's gaat en dat het een array is. Je kan hier bv ook gewoon `numberA` als naam gebruiken.